### PR TITLE
Adding py310 to manywheel images

### PR DESCRIPTION
**Pull request by atalman**
Adding py310 to manywheel images

### CODE REVIEW

**Review by janeyx99 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: .github/workflows/build-manywheel-images.yml [hunk 1]
```diff
@@ -33,6 +33,10 @@ jobs:
       GPU_ARCH_TYPE: cuda
       GPU_ARCH_VERSION: ${{ matrix.cuda_version }}
     steps:
+      - name: "[FB EMPLOYEES] Enable SSH (Click me for login details)"
+        uses: seemethere/add-github-ssh-key@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (malfet) [inline .github/workflows/build-manywheel-images.yml:39]: Nit: can you please move it into a separate PR (as it has nothing to do with feature in question)

File: common/install_cpython.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
 GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
 
 # Python versions to be installed in /opt/$VERSION_NO
-CPYTHON_VERSIONS=${CPYTHON_VERSIONS:-"3.6.6 3.7.5 3.8.1 3.9.0"}
+CPYTHON_VERSIONS=${CPYTHON_VERSIONS:-"3.10.1"}
```
Comment (malfet) [inline common/install_cpython.sh:8]: Err, you not supposed to land it like that, do you?

**Review by malfet — CHANGES_REQUESTED** [review]
Please do not delete older CPYTHON_VERSIONS

File: common/install_cpython.sh (hunk 2 above)
Comment (atalman) [reply common/install_cpython.sh:8]: These are tests. basically to save some time

File: .github/workflows/build-manywheel-images.yml (hunk 1 above)
Comment (atalman) [reply .github/workflows/build-manywheel-images.yml:39]: Sure I am planning to remove this change, since it seems not to work now

**Comment by malfet** [comment]
@atalman if this is an experimental PR, can you please move it to `Draft` mode?
